docs/data-model.md

```diff
@@ -0,0 +1,76 @@
+# Data model
+
+> [!NOTE]
+> This is a _draft_ document. It will change as we develop our understanding of the services that this application encapsulates.
+
+This application will power two user-facing services: 'Manage school placements' and 'Claim funding for mentors' (a.k.a. Track & Pay). Some data entities will be used by both services – for example, Mentors, Providers and Schools. Others will only be relevant to one service – for example, Funding Claims and School Placements.
+
+## Entity Relationship Diagram (ERD)
+
+This diagram represents our current understanding of the data models that will exist within this application.
+
+There are a few things to bear in mind when reading this:
+
+- This diagram attempts to bridge the gap between a 'high level' list of entities, and a 'low level' database schema. It sits somewhere in between.
+- Attribute names have been written in `Pascal_Case_With_Underscores`. This is to aid readability because spaces are not allowed in Mermaid JS ERD attribute names. It does not represent the format of eventual database column names.
+- It is incomplete. As we continue developing our services, this diagram will undoubtedly change and grow.
+
+```mermaid
+erDiagram
+  Placement {
+    uuid Placement_ID PK
+    uuid Mentor_ID FK
+    uuid Trainee_ID FK
+    uuid School_ID FK
+    date Start_Date
+    date End_Date
+  }
+
+  School {
+    uuid School_ID PK
+    string URN FK
+  }
+
+  Mentor {
+    uuid Mentor_ID PK
+    string TRN FK
+  }
+
+  Provider {
+    uuid Provider_ID PK
+    string Accredited_Provider_ID FK
+  }
+
+  TrainingCompleted {
```

Comment:
`TrainingCompleted` infers a state of "completed". Do we want to close off this model to that extent right now? Would `Training` with a `completed_at` timestamp be more extendable?

Reply:
I tend to agree that `Training` would be a better name -- it works better a plural, too, ie a mentor has many `trainings` rather than `training_completeds`.

Reply:
same

Reply:
Yep – good point. The naming of this entity was something we collectively struggled with. I agree that "Completed" is a bit of a clunky work and doesn't really represent the meaning of this data.

I wanted to avoid the ambiguity of using the name "Training" on its own. My concern is that, if you picked this app up without prior context, you could easily mistake "Training" as meaning "Teacher Training". I feel like this is especially likely given we operate within ITT (Initial Teacher Training).

What do you think about "Mentor Training" as a model name, to avoid any ambiguity?
@Nitemaeric @elceebee @SocFoot